PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/common/task/IntellijAzureTaskManager.java

```diff
@@ -62,7 +62,7 @@ protected void doRunAndWait(final Runnable runnable, final AzureTask<?> task) {
 
     @Override
     protected void doRunInBackground(final Runnable runnable, final AzureTask<?> task) {
-        final Task.Backgroundable backgroundTask = new Task.Backgroundable((Project) task.getProject(), task.getTitle(), task.isCancellable()) {
+        final Task.Backgroundable backgroundTask = new Task.Backgroundable((Project) task.getProject(), task.getTitle().getTitle(), task.isCancellable()) {
```

Comment:
getOpeation().getTitle()?

Reply:
Minister seconded.

Reply:
getTitle().getTitle() is strange.

Reply:
I have the same feeling when naming this method.
> getOpeation().getTitle()?
task itself is a subtype of operation.